scripts/infra/cloud-instance.sh

```diff
@@ -0,0 +1,399 @@
+#!/bin/bash -e
```

Comment:
```suggestion
# SPDX-License-Identifier: Apache-2.0

#!/usr/bin/env -S bash -e
```

Reply:
Note the shebang has to be the first line.